src/main/java/jenkinsci/plugins/influxdb/generators/SonarQubePointGenerator.java

```diff
@@ -49,29 +48,34 @@
     private static final short DEFAULT_MAX_RETRY_COUNT = 10;
     private static final int DEFAULT_RETRY_SLEEP = 5000;
 
-    private static final String URL_PATTERN_IN_LOGS = ".*" + Pattern.quote("ANALYSIS SUCCESSFUL, you can browse ")
-            + "(.*)";
-    private static final String TASK_ID_PATTERN_IN_LOGS = ".*" + Pattern.quote("More about the report processing at ")
-            + "(.*)";
+    // patterns used for data extraction from ${WORKSPACE}/${SONAR_BUILD_REPORT} file
+    private static final String SONAR_BUILD_REPORT = "/build/sonar/report-task.txt";
```

Comment:
`.scannerwork` is the default directory for SonarQube analysis
```suggestion
    private static final String SONAR_BUILD_REPORT = "/.sonarscanner/report-task.txt";
```

Reply:
Not relevant with the new changes.